<launch>
	<node pkg ="android_server" name = "android_server" type="a_server.py" output = "screen">
		<param name ="interface" value = "wlan0"/>
		<param name ="port"	 value = "8000"/>	
	</node>
</launch>
